### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by Timowest12**
Refractored to use classes and methods instead of functions

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js.js [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
+      booklist = JSON.parse(localStorage.getItem('books'));
+    }
+    booklist.push(this);
+    localStorage.setItem('books', JSON.stringify(booklist));
+    outputbooklist();
+  }
+
+  removefromlist(index) {
+    const bookss = JSON.parse(localStorage.getItem('books'));
+    bookss.splice(index, 1);
```
Comment (mikenath223) [inline js.js:33]: - [x] The localstorage functionality doesn't work as expected, it should still retain the same number of books after reloading the page. Kindly fix it.

Comment (henatan99) [reply js.js:33]: This issue is well addressed. Nice job! :pray:

File: js.js [hunk 2]
```diff
@@ -26,15 +44,15 @@ const addBooks = () => {
...
-
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
+  const newBook = new Book(title, author).addtolist();
+  bttnn.addEventListener('click', newBook);
   outputbooklist();
 };
+function removefromlist(index) {
+  const newBook = new Book().removefromlist(index);
+  bttnn.addEventListener('click', newBook);
```
Comment (henatan99) [inline js.js:53]: - [ ] OPTIONAL: This function is throwing a console error. Kindly incorporate error handling here as your app needs to be free of errors even though the app is working. :pray: In addition, I don't think you need to initialize a new book object as you remove a book from a list using its index. :pray: 
Screenshot from 2021-10-19 17-52-21

**Review by henatan99 — APPROVED** [review]
Hi @Timowest12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**